progressview/src/main/java/com/skydoves/progressview/HighlightView.kt

```diff
@@ -132,12 +140,21 @@ class HighlightView(
       if (orientation == ProgressViewOrientation.VERTICAL) {
         gradientOrientation = GradientDrawable.Orientation.TOP_BOTTOM
       }
-      GradientDrawable(
-        gradientOrientation,
-        intArrayOf(colorGradientStart, colorGradientEnd)
-      ).apply {
-        applyRadius(this)
-      }
+            if (colorGradientCenter == NO_COLOR) {
+                GradientDrawable(
+                    gradientOrientation,
+                    intArrayOf(colorGradientStart, colorGradientEnd)
+                ).apply {
+                    applyRadius(this)
+                }
+            } else {
+                GradientDrawable(
+                    gradientOrientation,
+                    intArrayOf(colorGradientStart, colorGradientCenter, colorGradientEnd)
+                ).apply {
+                    applyRadius(this)
+                }
+            }
```

Comment:
How about this way?

```kotlin
GradientDrawable(
        gradientOrientation,
        intArrayOf(colorGradientStart, colorGradientCenter, colorGradientEnd)
          .filter { it != NO_COLOR }.toIntArray()
      ).apply {
        applyRadius(this)
      }
```

Reply:
Because I want to do like this picture (yellow progress bar), I want add color in gradient center .
![image](https://user-images.githubusercontent.com/6047643/123532446-a836b080-d73f-11eb-8dd2-9a2386db9a36.png)

Reply:
Yes, exactly. So I added `colorGradientCenter` on the int array and filtered if the color value is `NO_COLOR` on my suggestion. 😄

Reply:
Oh! I understand what you said, right.

Reply:
Update at commit b7e5050b443912a156fff5bcaa4d0115614fb19e